themes/wilson_theme_starterkit/.storybook/wilsonStorybookTheme.js

```diff
@@ -0,0 +1,8 @@
+import { create } from '@storybook/theming/create';
```

Comment:
One of the actions the developer would have to do once they install Wilson is to clone and rename the theme to whatever client they're working on. Soon after the site is setup, the fact that it came from Wilson will pretty much be hidden (as it's totally unimportant past installation). So I'm thinking we should be agnostic with the name with all this stuff. Maybe just run with 'Storybook'?

Reply:
I've changed the theme to exampleStorybookTheme

You don't need one at all, but I've left this in, to at least demo having an alternative logo and speed up any customisation we may do 👍 . Agreed that naming it more generically is best.